cloudevents-sdk-warp/src/lib.rs

```diff
@@ -0,0 +1,254 @@
+#[macro_use]
+mod headers;
+
+mod server_request;
+mod server_response;
+
```

Comment:
I usually prefer to re-export here with a `pub use` the `request_to_event` and `event_to_response` methods, in order to give greater flexibility to the users

Reply:
I don't think this is a good idea, as we still learn `warp` api we want to keep internal methods private so we can remove them without affecting users. It is easier adding methods later than removing it once they are public. 

Also, this private part may be relevant for `hyper` support so it may be extracted to different crate.

My suggest is to keep this private at the moment

Reply:
Ok I'm fine with that at this point, if that's needed we'll make them public later